app/components/title-bar/title-bar.tsx

```diff
@@ -18,18 +17,14 @@ export const TitleBar: FC = () => {
   const el = document.querySelector('.draggable-bar');
   const location = useLocation();
 
-  const { backUrl } = useContext(BackContext);
   const routerHistory = useHistory();
-  const handleHistoryBack = () => {
-    if (backUrl === null) return;
-    routerHistory.push(backUrl);
-  };
+
   const winState = useWindowFocus();
 
   const { isNewerReleaseAvailable, latestRelease } = useCheckForUpdates();
 
   const isOnboarding = matchPath(location.pathname, { path: '/onboard' }) !== null;
-  const dulledTextColor = winState === 'blurred' ? '#A1A7B3' : undefined;
+  const dulledTextColor = winState === 'blurred' ? '#a1a7b3' : undefined;
```

Comment:
https://github.com/blockstack/stacks-wallet/pull/534/files#diff-cc3e923d8dd711bd6ff3d359583260ef2d9773b706bb0ff9e022100b7c2f8ca1R35 has a different color, should they be the same?

Reply:
These are different I'm pretty sure.